__tests__/app.test.js

```diff
@@ -13,3 +25,26 @@ describe("GET /api", () => {
       });
   });
 });
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topics", () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then((response) => {
+      const topics = response.body.topics
+        expect(topics.length).toBe(3);
+          topics.forEach((topic) => {
+          expect(typeof topic.description).toBe("string");
```

Comment:
Consider practicing using .toMatchObject here and the expect.any() statements for these types of assertions :) - this works though!